xmipptomo/protocols/protocol_apply_alignment_subtomo.py

```diff
@@ -31,22 +31,18 @@
 from pwem.emlib.image import ImageHandler
 from pwem.protocols import EMProtocol
 import pwem.emlib.metadata as md
-
 from pyworkflow.protocol.params import PointerParam
 
 # Tomo plugin must be installed
-AverageSubTomogram = pwem.Domain.importFromPlugin('tomo.objects',
-                                                  'AverageSubTomogram',
-                                                  doRaise=True)
-xmippToLocation = pwem.Domain.importFromPlugin('xmipp3.convert',
-                                               'xmippToLocation')
-writeSetOfVolumes = pwem.Domain.importFromPlugin('xmipp3.convert',
-                                                 'writeSetOfVolumes')
-alignmentToRow = pwem.Domain.importFromPlugin('xmipp3.convert',
-                                              'alignmentToRow')
+AverageSubTomogram = pwem.Domain.importFromPlugin('tomo.objects', 'AverageSubTomogram', doRaise=True)
```

Comment:
Since these dependencies are "hard" I'd have here normal imports.
and add a requirement to scipion-em-tomo in requirements.txt